src/Area_de_compras/Boleteria.java

```diff
@@ -58,15 +58,21 @@ public void comprarBoleto(int ci, MetodoPago metodo, String codigoSala,
         String fullNameClient = cliente.getFullName();
 
         if (cliente.getCantidadTicketsGratis() == 0){
+
             Boleto boleto = new Boleto(cliente, precioPelicula, salaCliente,pelicula);
             boleto.aplicarDescuento(metodo, dia);
             manager.buySeats(codigoSala, seatsToBuy, fullNameClient);
-        }else {
+        }
+
+        else {
+
             // Este ticket ganado por premio no te permite ganar mas puntos.
             Boleto boleto = new Boleto(salaCliente);
             manager.buySeats(codigoSala, seatsToBuy, fullNameClient);
             System.out.println("Su ticket ha sido utilizado.");
             cliente.usarTicketGratis();
+            precioTotal=((int)boleto.precioPelicula*seatsToBuy.length);
```

Comment:
```suggestion
            precioTotal = (int) (boleto.precioPelicula * seatsToBuy.length);
```
Tambien comenta que es el precio final del boleto procesado